js/components/dashboard/question-details.js

```diff
@@ -0,0 +1,53 @@
+import React, { PureComponent } from 'react'
+import { Modal } from 'react-bootstrap'
+import Button from '../common/button'
+import ReportQuestionDetails from '../report/question-details'
+import AnswersTable from '../../containers/report/answers-table'
+import css from '../../../css/dashboard/question-details.less'
+
+export default class QuestionDetails extends PureComponent {
+  constructor (props) {
+    super(props)
+    this.state = {
+      answersVisible: false
+    }
+    this.toggleAnswersVisibility = this.toggleAnswersVisibility.bind(this)
+  }
+
+  toggleAnswersVisibility () {
+    this.setState({answersVisible: !this.state.answersVisible})
+  }
+  render () {
+    const { selectedQuestion, onClose } = this.props
+    const { answersVisible } = this.state
+    const answers = selectedQuestion && selectedQuestion.get('answers')
+      ? selectedQuestion.get('answers').sortBy(a =>
+        (a.getIn(['student', 'lastName']) + a.getIn(['student', 'firstName'])).toLowerCase())
+      : []
+    const prompt = selectedQuestion && selectedQuestion.get('prompt')
+    return (
+      <Modal
+        show={selectedQuestion.size > 0}
+        onHide={onClose}
+        contentLabel='Example Modal'
```

Comment:
Not sure if we want "Example modal" label. :wink: And actually I've seen some warning in dev console re contentLabel prop.